### DISCUSSION TITLE
Increase Wind Turbine Penetration in to grid

### DISCUSSION THREAD

**Discussion by Mishal2024** [CURENT/andes#488] [Q&A]
Dr. Hantao,

I have an inquiry.

Is there any script to use like I see in Kundur's examples to increase wind turbine penetration in ANDES?

My goal is to observe the effects of wind turbine participation in grid frequency response. Incorporating Wind Turbine Models into grid.

**Comment by cuihantao** [comment]
Hi Mishal,

You can start with other IEEE 14 examples that have renewables, such as `ieee14_wt3.xlsx`. Increase wind turbine penetration by replacing synchronous generators with renewable energy models.
